drafts/Stormless2.md

```diff
@@ -0,0 +1,266 @@
+# Tutorial: Building a community sign-up app with Serverless, StepFunctions, and StackStorm Exchange - Episode 2
+
+
+Continue from [episode one][ep-one] to build a serverless app on AWS with [Serverless Framework][sf], AWS Lambda, StepFunctions, DynamoDB, API Gateway, and ready-to-use functions from StackStorm Exchange open-source catalog.
+
+[sf]:(https://serverless.com/framework)
+
+In this short episode, I'll add two more actions: create a contact in ActiveCampaign, and record a user in Dynamo DB. The full code for this episode is on Github at [2-add-more-actions branch](https://github.com/dzimine/slack-signup-serverless-stormless/tree/DZ/2-add-more-actions).
+
+> If you missed [Episode One][ep-one], please go there. It will be the right place to start, whether you are following along, or just skimming over to grasp the idea. 
+
+[ep-one]:(https://medium.com/@dzimine/tutorial-building-a-community-on-boarding-app-with-serverless-stepfunctions-and-stackstorm-b2f7cf2cc419)
+
+### Add more actions
+
+To continue to the next step, you'll need an [ActiveCampaign](https://www.activecampaign.com) account. Fear not: it takes 2 minutes - I just tried - they didn't ask me for a credit card or anything stupid. Just email. Once in, go to "My Settings", and select "Developer". 
+
+<TODO: add screen shot> 
+
+Note the `URL` and `Key` fields in the "API access" section. Add them into `env.yml` like this:
+
+```
+# Copy to env.yml and replace with your values.
+# Don't commit to Github :)
+slack:
+  admin_token: "xoxs-111111111111-111111111111-..."
+  organization: "your-team"
+active_campaign:
+  url: "https://YOUR-COMPANY.api-us1.com"
+  api_key: "1234567a9012a12345z12aa2..."
+
+```
+
+
+> PRO TIP: If you're not in the mood for Active Campaign, or any sign-ups, just mock up the API endpoint with something like [Mocky](https://www.mocky.io/) or [mockable.io](https://www.mocky.io/) and adjust the examples accordingly. For bonus points, create another serverless function with API Gateway endpoint in your `serverless.yml` and use it to mock the ActiveCampaign API call. 
```

Comment:
Mocking via another lambda looks neither easier nor faster. I'd rather avoid mentioning it at all.